favorite_foods.md

```diff
@@ -8,3 +8,4 @@
 - Fish
 - Cucumber
 - Chicken Tenders
+- Japanese curry chicken
```

Comment:
I didn't know you could grow curry chickens

Reply:
i love japanese curry

Reply:
![](https://media.giphy.com/media/l0ExayQDzrI2xOb8A/giphy.gif)